types/react-select/test/examples/CustomNoOptionsMessage.tsx

```diff
@@ -18,7 +18,7 @@ const NoOptionsMessage = (props: any) => {
 export default class CustomNoOptionsMessage extends React.Component {
   render() {
     return (
-      <Select
+      <Select<{ label: string; value: string }>
```

Comment:
I'm not sure why you have changed this but none of the other places where Select is used without its type parameter - is there a reason?

Reply:
Documented.

Reply:
Maybe it's obvious but not sure where you documented this? (on my phone so maybe missing something...)

Reply:
I added a comment in the source code above this line.  The "Conversation" tab seems to show the revision of the file you originally commented on, but the "Files" tab shows the latest revision [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30730/files#diff-89b26201789aeff667fae9d3c31d3e75R21).